packages/MiniExpandCollapse/MiniExpandCollapse.jsx

```diff
@@ -70,6 +83,10 @@ MiniExpandCollapse.propTypes = {
 
 MiniExpandCollapse.defaultProps = {
   expandTitle: null,
+  onToggle: () => {
+    // eslint-disable-next-line no-console
+    console.log('Hello World')
```

Comment:
I just want to be sure I am understanding correctly, if no onToggle function is passed, in production every time a user clicks to toggle open we will be logging 'Hello World'?

Reply:
Hi @janeszelag. If no onToggle is passed, then the default value for the prop should be `() => {}`. I will remove the console.log statement. I will also add an example to the new `onToggle` prop to `MiniExpandCollapse.md`.